From cded0460851f17717789b2ddcad2595660e9106d Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Tue, 8 May 2018 14:09:45 +0200
Subject: [PATCH 10/11] lineage: Enable BOARD_PROPERTY_OVERRIDES_SPLIT_ENABLED

 * Our properties were supposed to go to /system/etc/prop.default
    after the following commit:
    "lineage: Move to Google's method of defining system default props"
    Change-Id: I6cb0e28a7599b010b389cc541015a37010a00f4b

 * However if BOARD_PROPERTY_OVERRIDES_SPLIT_ENABLED is not true,
    only /default.prop will contain the contents of
    ADDITIONAL_DEFAULT_PROPERTIES and PRODUCT_SYSTEM_DEFAULT_PROPERTIES

 * Enabling BOARD_PROPERTY_OVERRIDES_SPLIT_ENABLED globally adds
    system/etc/prop.default and vendor/default.prop properly

Change-Id: I6975960a83db0f61b57db0e145e9d51068e71360
---
 config/BoardConfigLineage.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/config/BoardConfigLineage.mk b/config/BoardConfigLineage.mk
index e097cceb..023e229e 100644
--- a/config/BoardConfigLineage.mk
+++ b/config/BoardConfigLineage.mk
@@ -3,6 +3,9 @@ ifeq ($(WITH_LINEAGE_CHARGER),true)
     BOARD_HAL_STATIC_LIBRARIES := libhealthd.lineage
 endif
 
+# Properties
+BOARD_PROPERTY_OVERRIDES_SPLIT_ENABLED := true
+
 # QCOM HW crypto
 ifeq ($(TARGET_HW_DISK_ENCRYPTION),true)
     TARGET_CRYPTFS_HW_PATH ?= vendor/qcom/opensource/cryptfs_hw
-- 
2.17.0

